diffrax/brownian/tree.py

```diff
@@ -70,10 +111,18 @@ def __init__(
         tol: Scalar,
         shape: Union[Tuple[int, ...], PyTree[jax.ShapeDtypeStruct]],
         key: "jax.random.PRNGKey",
+        levy_area: str = "",
     ):
+        if t0 >= t1:
+            raise ValueError("t0 must be strictly less than t1.")
```

Comment:
If they're not JAX arrays then I the `t0 >= t1` condition should be evaluated statically, in which case the `error_if` will eagerly raise at trace-time rather than runtime. Thus it should always work :)